code/modules/blacksmithing/anvil.dm

```diff
@@ -42,3 +42,6 @@
 		if(W.loc == src.loc && params)
 			W.setPixelOffsetsFromParams(params, user)
 			return 1
+
+/obj/item/anvil/attack_tk(mob/user)
+	return attack_hand(user)
```

Comment:
Won't this remotely grab the anvil and put it in your hand

Reply:
Already tested it plenty, It's directly copied from some button code to prevent TK users too.